src/main/groovy/org/edx/jenkins/dsl/DevopsConstants.groovy

```diff
@@ -93,11 +93,6 @@ class DevopsConstants {
             if (branchName == 'master') {
                 githubPush()
             }
-            // for all other jobs in this style, poll github for new commits on
-            // the 'defaultBranch' every 10 minutes
-            else {
-                scm('H/10 * * * *')
-            }
```

Comment:
We don't want to support polling. Jobs that can have webhooks (i.e. applications with application code repositories) have webhooks. Others (e.g. firefox, chrome, rabbitmq, xenial-common) will either be triggered by configuration-watcher or manually.